lib/bots/versionbot.ts

```diff
@@ -448,6 +458,50 @@ export class VersionBot extends ProcBot {
 	}
 
 	/**
+	 * Always passes Versionist and AutoMerge status if a no-checks label has been applied.
+	 * This allows manual merging to correctly go ahead.
+	 *
+	 * @param _registration  GithubRegistration object used to register the method
+	 * @param event          ServiceEvent containing the event information ('pull_request' event)
+	 * @returns              A void Promise once execution has finished.
+	 */
+	protected passWithNoChecks = (_registration: GithubRegistration, event: ServiceEvent): Promise<void> => {
+		const pr = event.cookedEvent.data.pull_request;
+		const head = event.cookedEvent.data.pull_request.head;
+		const owner = head.repo.owner.login;
+		const repo = head.repo.name;
+		const prNumber = pr.number;
+
+		this.logger.log(LogLevel.INFO, `Skipping AutoMerge and Versionist checks for ${owner}/${repo}#${prNumber}`);
+
+		// Always set the automerge and versionist status to passing.
+		return Promise.join([
+			this.dispatchToEmitter(this.githubEmitterName, {
+				data: {
+					context: StatusAutoMerge.Context,
+					description: StatusAutoMerge.ManualMerge,
+					owner,
+					repo,
+					sha: head.sha,
+					state: 'success'
+				},
+				method: this.githubApi.repos.createStatus,
+			}),
+			this.dispatchToEmitter(this.githubEmitterName, {
+				data: {
+					context: StatusVersionist.Context,
+					description: StatusVersionist.NoVersioning,
+					owner,
+					repo,
+					sha: head.sha,
+					state: 'success'
+				},
+				method: this.githubApi.repos.createStatus,
+			})
+		]).return();
```

Comment:
This is a little confusing, you have a `Promise.join`, but no handling function. Should this not be a `Promise.all`?